ucf_exchange_client/connection.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+import websockets
+
+from .RPC_types import msg_from_json, msg_to_json
```

Comment:
so I just changed this functionality on `master`, you should `import json` and `json.dumps` the message yourself.